### DISCUSSION TITLE
platform api does not work with newly issued platform access key

### DISCUSSION THREAD

**Discussion by dearkafka** [chatwoot/chatwoot#6607] [Chatwoot APIs]
So, I have a self-hosted chatwoot. I created a platform app via Super-user. I have obtained access key for a platform. I'm trying to access this `platform/api/v1/accounts/1/account_users` or this `platform/api/v1/accounts/2/account_users` (created via web), but I always got  `body={'error': 'Non permissible resource'} ` - 401. why? nginx underscore is set on.

to clarify, nginx correctly pass headers. user specific token works fine on user specific api methods.

using latest docker image and compose https://github.com/eremeye/helpdesk-workshop/blob/master/chatwoot/docker-compose.yml

**Comment by dearkafka** [comment]
Also, looking into console (I dont know ruby so sorry for mistakes if any)
```

app = PlatformApp.create(name: 'name')
a = AccessToken.find_by(token: app.access_token.token)
```
and then
```

a.owner
=>
#<PlatformApp:0x00007f2a3944d8c8
 id: 2,
 name: "name",
 created_at: Sat, 04 Mar 2023 03:04:55.528756000 UTC +00:00,
 updated_at: Sat, 04 Mar 2023 03:04:55.528756000 UTC +00:00>
```
but
```

a.owner.platform_app_permissibles
=> []

@resource = Account.find(1)
a.owner.platform_app_permissibles.find_by(permissible:@resource)
=> nil
```

**Comment by dearkafka** [comment]
@pranavrajs so, regarding #6611 , how can I access other accounts in system? if they are not created by that platform? there is no way to do it through api, right?

**Reply by pranavrajs** [reply]
No, it is not possible.

**Reply by dearkafka** [reply]
thanks, noted

**Reply by dearkafka** [reply]
btw is this documented? I feel like I missed it

**Reply by pranavrajs** [reply]
@dearkafka, I'm not sure. Let me check it out and see if we can improve that. Thanks for the note.
